app/app.go

```diff
@@ -453,3 +459,20 @@ func GetMaccPerms() map[string][]string {
 
 	return dupMaccPerms
 }
+
+// Required for ibctesting
+func (app *OsmosisApp) GetStakingKeeper() stakingkeeper.Keeper {
+	return *app.AppKeepers.StakingKeeper
+}
+
+func (app *OsmosisApp) GetIBCKeeper() *ibckeeper.Keeper {
+	return app.AppKeepers.IBCKeeper
+}
+
+func (app *OsmosisApp) GetScopedIBCKeeper() capabilitykeeper.ScopedKeeper {
+	return app.AppKeepers.ScopedIBCKeeper
+}
```

Comment:
Lets move these to modules.go, we have to do something similar for account keeper

Reply:
done!